### PR TITLE
Fixes Vape Pen (e-cig) reagent spam

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
fix: Vape Pens (e-cigs) will now consume reagents proportional to the vape size and static smoke production. 
/:cl:

Vape pens were shitting out 4x their total reagent contents every few tics, essentially pooping out hundreds of reagents as smoke while consuming only a tiny amount. 

This, by pure accident, means you can no longer create blood or water pools from fucking vaping.

closes #35177

### CODE REVIEW

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -785,14 +785,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
...
-		var/datum/effect_system/smoke_spread/chem/s = new
-		s.set_up(reagents, 1, loc, silent=TRUE)
+		var/datum/effect_system/smoke_spread/chem/smoke_machine/s = new
+		s.set_up(reagents, 1, 24, loc)
 		s.start()
 		vapetime = 0
 
 	if((obj_flags & EMAGGED) && vapetime > 3)
-		var/datum/effect_system/smoke_spread/chem/s = new
-		s.set_up(reagents, 4, loc, silent=TRUE)
```
Comment (PKPenguin321) [inline code/game/objects/items/cigs_lighters.dm]: removing silence means these will spam admins to hell and back

Comment (Robustin) [reply code/game/objects/items/cigs_lighters.dm]: The smoke machine smoke setup doesn't have admin notifications for the actual smoke (to remove spam), so this shouldn't either.

Comment (PKPenguin321) [reply code/game/objects/items/cigs_lighters.dm]: ???
just leave the silent arg, do you know what it does?

**Comment by Bawhoppen** [comment]
Can't go more green than synthesizing chemicals out of thing air.

**Comment by improvedname** [comment]
Looks like oldman cared about vapes
Big shocker it involved smoke.

**Comment by ExcessiveUseOfCobblestone** [comment]
damn does DGL not have a github i wanna @ him

**Comment by Iamgoofball** [comment]
:+1:

**Pull request merged**